cloud/scope/cluster.go

```diff
@@ -237,6 +237,7 @@ func (s *ClusterScope) NatRouterSpec() *compute.Router {
 				Name:                          fmt.Sprintf("%s-%s", networkSpec.Name, "nat"),
 				NatIpAllocateOption:           "AUTO_ONLY",
 				SourceSubnetworkIpRangesToNat: "ALL_SUBNETWORKS_ALL_IP_RANGES",
+				MinPortsPerVm:                 s.GCPCluster.Spec.Network.MinPortsPerVM,
```

Comment:
So how does defaulting work here with existing objects?  I guess worst-case (default not applied) the code does the same thing it did before (and 64 is also the default on the NatRouter, I believe, so we shouldn't be changing the actual value in GCP either)